lib/machine/defaultAnalyzerFactory.ts

```diff
@@ -55,9 +55,6 @@ export const defaultAnalyzerFactory: AnalyzerFactory = sdm => {
         .withScanner(k8sScanner)
         .withScanner(travisScanner)
         .withScanner(preferencesScanner)
-        .withScanner(dockerScanner)
-
-        .withInterpreter(new DockerfileInterpreter())
```

Comment:
We need this so that we have one place to add the immaterial change test for the Dockerfile. Or we have to replicate that push test in every Interpreter. In any way this change can't go out as it would not trigger goals for simple changes to the Dockerfile.

Reply:
Done. Made a DockerStack.